README.md

```diff
@@ -26,6 +26,44 @@ add as much states as preferred, and to create custom natural history of the dis
 
 For more information, please refer to the official documentation of the project.
 
+## Testing
+
+All the test were developed with the python tool pytest. To see them properly, type 
+in the command line: 
+
+`$ pytest -s -v <name_of_test>`
```

Comment:
Change to use code block syntasis. See https://www.freecodecamp.org/news/how-to-format-code-in-markdown/